### PR TITLE
Feature - Update card boxes to be full height wherever used

### PR DESCRIPTION
**Pull request by MoiyadKayda**
## Screenshots:
image

## Checklist:

- [ ] I have updated the CHANGELOG with a summary of my changes

### CODE REVIEW

File: src/packages/core/src/sections/testimonialBoxes1.tsx [hunk 1]
```diff
@@ -57,13 +57,15 @@ export const TestimonialBoxes1 = (props: ITestimonialBoxes1Props): React.ReactEl
...
                 <Box key={index} variant={boxVariant} isFullHeight={boxVariant !== 'card'}>
                   <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} isFullWidth={true}>
-                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} isFullWidth={true} isFullHeight={true} shouldAddGutters={true}>
-                      { getIcon(box.type, box.iconId, box.iconColor) }
-                      <Stack.Item growthFactor={1} shrinkFactor={1}>
-                        {box.url && <Link target={box.url} text={box.author} />}
-                        {!box.url && <Text variant='bold'>{box.author}</Text>}
-                      </Stack.Item>
-                    </Stack>
+                    <Stack.Item growthFactor={1} shrinkFactor={1}>
```
Comment (krishan711) [inline src/packages/core/src/sections/testimonialBoxes1.tsx]: dont need this i think

File: src/packages/core/src/sections/testimonialBoxes1.tsx [hunk 2]
```diff
@@ -57,13 +57,15 @@ export const TestimonialBoxes1 = (props: ITestimonialBoxes1Props): React.ReactEl
               {props.boxes.map((box: ITestimonialBoxes1Box, index: number): React.ReactElement => (
                 <Box key={index} variant={boxVariant} isFullHeight={boxVariant !== 'card'}>
```
Comment (krishan711) [inline src/packages/core/src/sections/testimonialBoxes1.tsx]: this is the only line that should change

File: src/packages/core/src/sections/testimonialBoxes1.tsx [hunk 3]
```diff
@@ -55,14 +55,12 @@ export const TestimonialBoxes1 = (props: ITestimonialBoxes1Props): React.ReactEl
             {props.subtitleText && <Stack.Item gutterAfter={PaddingSize.Wide2}><SectionSubtitleText text={props.subtitleText} /></Stack.Item>}
             <EqualGrid childAlignment={Alignment.Fill} shouldAddGutters={true} childSizeResponsive={{ base: 12, small: 12, medium: 6, large: 4 }}>
               {props.boxes.map((box: ITestimonialBoxes1Box, index: number): React.ReactElement => (
-                <Box key={index} variant={boxVariant} isFullHeight={boxVariant !== 'card'}>
+                <Box key={index} variant={boxVariant} isFullHeight={true}>
                   <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} isFullWidth={true}>
-                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} isFullWidth={true} isFullHeight={true} shouldAddGutters={true}>
+                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Start} isFullWidth={true} isFullHeight={true} shouldAddGutters={true}>
```
Comment (krishan711) [inline src/packages/core/src/sections/testimonialBoxes1.tsx]: all the changes below and including this line look suspcious - just to be clear nothing visual should change except the height of the box right?

File: src/packages/core/src/sections/testimonialAvatars1.tsx [hunk 4]
```diff
@@ -40,19 +40,17 @@ export const TestimonialAvatars1 = (props: ITestimonialAvatarProps): React.React
...
                     </Stack.Item>
                     <Stack direction={Direction.Horizontal} shouldAddGutters={true} childAlignment={Alignment.Center}>
                       {box.authorImageUrl && <Box width='50px' height='50px'><LazyImage isFullHeight={true} isFullWidth={true} variant={'profile'} source={box.authorImageUrl} alternativeText={box.authorName ? `${box.authorName} image` : 'Testimonial author image'} fitType='crop' /></Box>}
-                      <Stack.Item growthFactor={1} shrinkFactor={1}>
-                        <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} contentAlignment={Alignment.Center}>
-                          {box.authorName && <MarkdownText textVariant={'bold'} textAlignment={TextAlignment.Left} source={box.authorName} />}
-                          {box.authorTitle && <MarkdownText textVariant={'small'} textAlignment={TextAlignment.Left} source={box.authorTitle} />}
-                        </Stack>
-                      </Stack.Item>
+                      <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} contentAlignment={Alignment.Center}>
```
Comment (krishan711) [inline src/packages/core/src/sections/testimonialAvatars1.tsx:50]: was the stack.item definitely not achievign anything here?

File: src/packages/core/src/sections/testimonialBoxes1.tsx (hunk 3 above)
Comment (MoiyadKayda) [reply src/packages/core/src/sections/testimonialBoxes1.tsx]: the alignment should be left

File: src/packages/core/src/sections/testimonialAvatars1.tsx (hunk 4 above)
Comment (MoiyadKayda) [reply src/packages/core/src/sections/testimonialAvatars1.tsx:50]: I checked, the stack.item had no effect

File: src/packages/core/src/sections/testimonialBoxes1.tsx [hunk 5]
```diff
@@ -55,14 +55,12 @@ export const TestimonialBoxes1 = (props: ITestimonialBoxes1Props): React.ReactEl
...
                       { getIcon(box.type, box.iconId, box.iconColor) }
-                      <Stack.Item growthFactor={1} shrinkFactor={1}>
-                        {box.url && <Link target={box.url} text={box.author} />}
-                        {!box.url && <Text variant='bold'>{box.author}</Text>}
-                      </Stack.Item>
+                      {box.url && <Link variant='left' target={box.url} text={box.author} />}
```
Comment (krishan711) [inline src/packages/core/src/sections/testimonialBoxes1.tsx]: is there a 'left' variant on Links? I havent seen it

**Review by krishan711 — APPROVED** [review]

**Pull request merged**